scrnaseq.Rmd

```diff
@@ -563,14 +568,16 @@ param$feature_filter = purrr::map(list_names(sc), function(s) {
 # Calculate percentage of counts in mitochondrial genes for each Seurat object
 sc = purrr::map(sc, Seurat::PercentageFeatureSet, pattern=param$mt, col.name="percent_mt", assay="RNA")
 
+## percent_ribo?
```

Comment:
Sure

Reply:
Okay I have changed the code as follows:

```
# Calculate percentage of counts in mitochondrial genes for each Seurat object
sc = purrr::map(sc, function(s) {
  mt_features = grep(pattern=param$mt, rownames(sc), value=TRUE)
  return(Seurat::PercentageFeatureSet(s, features=mt_features, col.name="percent_mt", assay="RNA"))
})

# Calculate percentage of counts in ribosomal genes for each Seurat object
sc = purrr::map(sc, function(s) {
  ribo_features = grep(pattern="^RP[SL]", rownames(sc), value=TRUE, ignore.case=TRUE)
  return(Seurat::PercentageFeatureSet(s, features=ribo_features, col.name="percent_ribo", assay="RNA"))
})
```

Ribo genes are based on what I found here: https://nbisweden.github.io/excelerate-scRNAseq/session-qc/Quality_control.html